migrations/1584604583726_blocks.js

```diff
@@ -1,4 +1,7 @@
 /** @param { import("node-pg-migrate").MigrationBuilder } pgm */
+
+const INDEX_METHOD = process.env.PG_IDENT_INDEX_TYPE;
```

Comment:
@rafaelcr and I discussed this configurable index type in the initial draft PR, and we thought it would be better to decide on an index type and remove the config. IIRC we decided a lot of these indexes should be reverted back to `btree` (the default) instead of `hash`, because postgres is currently very slow with writing `hash` index types, and it's slowing down block ingestion.

Reply:
@zone117x Thanks for this note. But, not sure if I get you. So, should we stick to the migrations present on `master/develop` branches? Tks!

Reply:
Yes, we should stick with the current ones, or explicitly change them back to btree if writing is too slow with `hash`